php/js.php

```diff
@@ -206,4 +207,9 @@
 	f.parentNode.insertBefore(j, f);
 })(window, document, 'script', 'dataLayer', '<?php echo $this -> params -> get('gtm_account'); ?>');
 </script>
+<?php endif;
+
+// Load GA4 head code
+if($this -> params -> get('ga4_account', '')) : ?>
```

Comment:
I wanted for this to be loaded the same way we load the ga code, by dynamically creating the <script> from js, for example:

var myScript = document.createElement('script');
var firstScript = document.getElementsByTagName('script')[0];
myScript.async = 1;
myScript.src = 'https://www.googletagmanager.com/gtag/js?id=' + ga4_account';
firstScript.parentNode.insertBefore(myScript, firstScript);